src/libdiod/test/lock.c

```diff
@@ -0,0 +1,282 @@
+/************************************************************\
+ * Copyright 2010 Lawrence Livermore National Security, LLC
```

Comment:
Not a big deal, but since this file is new, should the copyright date be updated?

Reply:
Ah yes, thanks!  Just pushed a fix and I'll set MWP.